File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    numbers=list(map(int, s.split()))
    print(numbers)
    low=min(numbers)
    high=max(numbers)
    return f"{low} {high}"
    # for num in numbers:
    #     int_list+=(int(num))
    # return "".join(min(int_list)+max(int_list))
    # answer = ''
    #low=min(number)
   # high=max(number)
   # answer+=low + high

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    vector<int> nums;
    string temp = "";
    for (char c:s){
        if (c==' '){
            nums.push_back(stoi(temp));
            temp="";
        }
        else{
            temp+=c;
        }
    }
    nums.push_back(stoi(temp));
    int min_val = nums[0];
    int max_val = nums[0];
    for (int num:nums){
        if(num>max_val) max_val = num;
        if(num<min_val) min_val = num;
    }
    return to_string(min_val) + " " + to_string(max_val);
}